### PR TITLE
Resource page UI revision

### PR DESCRIPTION
**Pull request by aarsli**

### CODE REVIEW

**Review by kunpai — APPROVED** [review]
Looks good to me 👍

**Review by helloparthshah — COMMENTED** [review]
Looks good but let's wait till after the presentation to merge it

**Review by Harshil2107 — APPROVED** [review]
looks good 👍

**Pull request merged**
